SEOKMINJUN/Invaders-SDP
# Kan 44 integrate achievement with collection screen

## What
- Clean up the messy code.
- Modify it so that the clear count is accumulated.
- Change the color of the achievement name if it has been completed at least once.

## Why
- Since achievements can be cleared again when the game is restarted, I thought it would be a bit lacking to just display 'Complete,' so I modified it to accumulate the achievement count.
- I liked the visual representation of completing achievements, so I made it so that if an achievement has been cleared at least once, the achievement name color changes.

## How to Test
1. Run the game.
2. Clear the achievement.
3. Go to the Collections tab and check.

## Screenshots

![image](https://github.com/user-attachments/assets/4d1947d2-0d8b-435e-9904-c5cccad094de)
